### PR TITLE
Enable TypeAlias check by default

### LINKED ISSUE #75: Compatibility with flake8 --extend-select
It doesn't work:
```
% python -m flake8 --select=Y091 tests/raise.pyi
tests/raise.pyi:4:9: Y091 Function body must not contain "raise"
tests/raise.pyi:7:9: Y091 Function body must not contain "raise"
% python -m flake8 --extend-select=Y091 tests/raise.pyi
% 
```

### LINKED ISSUE #86: Revisit disabled-by-default errors
I'd like to get rid of the disabled-by-default Y090, Y091, and Y092. This will require a couple of different changes both in typeshed and here, so I'm opening this issue to track them all.

1. Y090 and Y091
These are essentially special cases of Y010 (don't put stuff in a function body) that we disabled for compatibility years ago. There are no remaining triggers for these rules in typeshed, so I propose that we get rid of the separate codes and make Y010 trigger on everything you may put in a function body.

2. Y092 triggers on a lot of existing typeshed code:

<details>

```
stdlib/typing_extensions.pyi:43:26: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:44:23: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:45:22: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:46:26: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:47:29: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:51:25: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:78:21: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:92:27: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:93:24: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:128:33: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:129:31: Y092 Top-level attribute must not have a default value
stdlib/typing_extensions.pyi:130:31: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:49:23: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:50:26: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:51:23: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:52:25: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:54:26: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:55:26: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:56:22: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:57:26: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:58:26: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:60:27: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:62:29: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:90:33: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:91:31: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:92:31: Y092 Top-level attribute must not have a default value
stdlib/typing.pyi:136:31: Y092 Top-level attribute must not have a default value
stdlib/@python2/asyncore.pyi:9:24: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing_extensions.pyi:37:26: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing_extensions.pyi:38:23: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing_extensions.pyi:42:25: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing_extensions.pyi:62:21: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing_extensions.pyi:73:27: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing_extensions.pyi:74:24: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing_extensions.pyi:103:29: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing_extensions.pyi:104:27: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing_extensions.pyi:105:27: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:25:23: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:26:26: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:27:23: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:28:25: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:29:26: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:30:26: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:31:22: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:32:26: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:33:23: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:39:25: Y092 Top-level attribute must not have a default value
stdlib/@python2/typing.pyi:41:21: Y092 Top-level attribute must not have a default value
stdlib/@python2/bdb.pyi:8:38: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:4:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:5:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:7:20: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:8:23: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:9:20: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:11:13: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:13:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:14:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:15:17: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:16:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:17:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:18:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:19:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:20:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:21:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:22:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:23:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:24:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:25:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:26:14: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:27:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:28:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:29:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:30:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:31:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:32:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:33:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:34:17: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:35:17: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:36:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:37:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:38:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:39:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:40:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:41:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:42:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:43:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:44:15: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:45:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:46:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:47:18: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:48:17: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:49:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:50:16: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:53:21: Y092 Top-level attribute must not have a default value
stdlib/@python2/signal.pyi:54:25: Y092 Top-level attribute must not have a default value
stdlib/@python2/ctypes/__init__.pyi:26:20: Y092 Top-level attribute must not have a default value
stdlib/@python2/ctypes/__init__.pyi:27:19: Y092 Top-level attribute must not have a default value
stdlib/@python2/ctypes/__init__.pyi:28:21: Y092 Top-level attribute must not have a default value
stdlib/@python2/ctypes/__init__.pyi:54:29: Y092 Top-level attribute must not have a default value
stdlib/@python2/ctypes/__init__.pyi:56:37: Y092 Top-level attribute must not have a default value
stdlib/@python2/ctypes/__init__.pyi:57:37: Y092 Top-level attribute must not have a default value
stdlib/@python2/ctypes/__init__.pyi:58:31: Y092 Top-level attribute must not have a default value
stdlib/@python2/ctypes/__init__.pyi:59:20: Y092 Top-level attribute must not have a default value
stdlib/@python2/dbm/ndbm.pyi:10:16: Y092 Top-level attribute must not have a default value
stubs/paramiko/paramiko/pkey.pyi:6:29: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:4:25: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:5:23: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:6:23: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:7:23: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:8:22: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:9:23: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:10:30: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:12:31: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:13:29: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:14:21: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:15:22: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:16:30: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:17:27: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:18:21: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:19:22: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:20:20: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:21:22: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:22:23: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:23:22: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:24:24: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:25:26: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:27:25: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:28:32: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:29:40: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:30:37: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:31:35: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:32:31: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:33:29: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:34:42: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:35:40: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:36:36: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:37:30: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:38:32: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:39:30: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:40:30: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:41:26: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:42:31: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:43:35: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:44:55: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:45:33: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:46:36: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:47:33: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:48:35: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:49:33: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:50:32: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:51:47: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:52:46: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:53:50: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:54:48: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:55:43: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:57:26: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:58:26: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:59:23: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:61:32: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:62:31: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:63:33: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:64:32: Y092 Top-level attribute must not have a default value
stubs/pyaudio/pyaudio.pyi:65:31: Y092 Top-level attribute must not have a default value
stubs/cryptography/cryptography/x509/oid.pyi:102:43: Y092 Top-level attribute must not have a default value
stubs/cryptography/cryptography/x509/oid.pyi:104:67: Y092 Top-level attribute must not have a default value
stubs/setuptools/pkg_resources/__init__.pyi:41:27: Y092 Top-level attribute must not have a default value
```

</details>

These are mostly of the form `constant: type = ...`, and they seem legitimate to me. I'll send a PR to typeshed to get rid of the `= ...`. After that we can consider turning it on by default.

### PR DESCRIPTION
**Pull request by JelleZijlstra**
And delete the machinery for disabling errors.

This made this check run on all tests, which helped me find a bug
(it triggered on __all__). I can split the test changes into a separate
PR if preferred.

I'll submit a separate PR to typeshed to disable this check for now.

Fixes #75. Fixes #86.

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
The typeshed-primer failures are all Y026. We'll have to merge python/typeshed#6958 first for CI to become green.

File: tests/attribute_annotations.pyi [hunk 1]
```diff
@@ -9,7 +8,6 @@ field7 = b""  # Y015 Attribute must not have a default value other than "..."
 class Foo:
     field1: int
     field2: int = ...
-    field3 = ...  # type: int
```
Comment (hauntsaninja) [inline tests/attribute_annotations.pyi:12]: If this triggered Y026, might be worth special casing `...`?

Comment (JelleZijlstra) [reply tests/attribute_annotations.pyi:12]: Yes, I can probably just exclude cases where the RHS is ellipsis. We should lint against this but Y026 isn't the right code.

**Review by hauntsaninja — APPROVED** [review]

**Review by AlexWaygood — COMMENTED** [review]
Looks good, but I think we should make sure we advertise this change more loudly, either in the README or the changelog (and maybe point to instructions elsewhere about how to disable certain codes in your flake8 config file?). If third-party users of this package are anything like typeshed, this check is going to cause a *large* number of hits.

**Comment by hauntsaninja** [comment]
Yeah, I have a tendency to think of typeshed as the main consumer of flake8-pyi, but that's not really the case.

It might be nice to collect the codemods we've applied to typeshed and link them in our CHANGELOG. It could also be nice to update stubgen output to be compliant with all these new lints.

**Comment by AlexWaygood** [comment]
> Yeah, I have a tendency to think of typeshed as the main consumer of flake8-pyi, but that's not really the case.
> 
> It might be nice to collect the codemods we've applied to typeshed and link them in our CHANGELOG. It could also be nice to update stubgen output to be compliant with all these new lints.

Linking to the PRs in the CHANGELOG sounds like a great idea.

We could maybe also have somewhere to collect some of the scripts we've used for codemodding — either a new repo, or a subdirectory in this repo. WDYT?

**Comment by hauntsaninja** [comment]
A scripts directory here seems like a decent place, at the very least we're all here now!

@hoefling shared an excellent libcst script in https://github.com/python/typeshed/pull/5564#issuecomment-896713040 that we could add on to.

**Comment by Akuli** [comment]
I don't like the idea of collecting everything into a `scripts` directory:
- We don't want to write tests for the scripts, or maintain them in general.
- We needed to write scripts for typeshed, because there's so many lines of stubs in typeshed. Most other stubs projects don't have anywhere near as much code, and should be find with manually editing the lines that this plugin points out.
- The scripts assume the code is blacked, have a few weird corner cases, and aren't exactly clean code. (This applies mostly to my scripts, though.)

I think it would be better to instead improve error messages to say what you should do instead. Some of our error messages already work that way:

```
Y019 Use "_typeshed.Self" instead of "_S", e.g. "def bad_instance_method(self: Self, arg: bytes) -> Self: ..."
```

This makes writing a script easy enough to just do yourself if you need one. We could even include one in this repo, and if it's simple enough, it shouldn't be too much work to test and maintain.

Improving stubgen output would also be nice :)

**Comment by AlexWaygood** [comment]
I wasn't envisioning writing tests for the scripts. We could have a README file in the subdirectory that has a big "USE WITH CAUTION — UNTESTED" notice in it.

I also didn't mean to suggest that we needed to include every codemodding script we'd ever written in the subdirectory. It could be a selection.

**Comment by JelleZijlstra** [comment]
> Looks good, but I think we should maybe advertise this change more loudly

I added a section listing codes users may not want to enable. Your Y027 can also get a mention there.

I'd be OK with adding cleanup scripts to this repo, but let's do that in a separate PR.

**Comment by AlexWaygood** [comment]
> I'd be OK with adding cleanup scripts to this repo, but let's do that in a separate PR.

agreed.

**Review by AlexWaygood — APPROVED** [review]
LGTM

**Pull request merged**